### DISCUSSION TITLE
TShock 4.5.15 for Terraria 1.4.3.5

### DISCUSSION THREAD

**Discussion by hakusaro** [Pryaxis/TShock#2575] [Show and tell]
## Notable changes
* Added preliminary support for Terraria 1.4.3.5. (@SignatureBeef, @hakusaro)

## Upgrading
To upgrade TShock, replace all of the files in the location of your server installation with all of the files from the release zip we provide. Overwrite all files. No data will be lost.

<hr /><em>This discussion was created from the release <a href='https://github.com/Pryaxis/TShock/releases/tag/v4.5.15'>TShock 4.5.15 for Terraria 1.4.3.5</a>.</em>

**Comment by razski** [comment]
It's a shame they updated Terraria again within 24 hours of this release

**Reply by hakusaro** [reply]
Mmm, no, we like them updating and fixing things!

**Reply by razski** [reply]
What I meant was it was irritating to need to update Tshock this morning, and already it's out of date.

**Comment by zvikasdongre** [comment]
Hi There, 
I am getting an error, should I create a new issue or post it here?

**Reply by hakusaro** [reply]
You should post a new discussion.

**Reply by zvikasdongre** [reply]
Here it is: [https://github.com/Pryaxis/TShock/discussions/2588](https://github.com/Pryaxis/TShock/discussions/2588), Can you please check it?
